caiman_napari/cnmfe_gui.py

```diff
@@ -8,20 +8,103 @@ def __init__(self, parent):
         QtWidgets.QDockWidget.__init__(self, parent=parent)
         self.ui = Ui_CNMFEDockWidget()
         self.ui.setupUi(self)
+        self.ui.btnAddToBatchCorrPNR.clicked.connect(self._add_item_corr_pnr)
+        self.ui.btnAddToBatchCNMFE.clicked.connect(self._add_item_cnmfe)
 
     @present_exceptions()
-    def get_params(self, *args, group_params: bool = True) -> Tuple[str, bool, dict]:
-        """
-        Get a dict of the set parameters.
-        If the work environment was loaded from a motion correction batch item it put the bord_px in the dict.
-        Doesn't use any arguments
+    def _add_item_corr_pnr(self, *args) -> Tuple[str, dict]:
+        gSig = self.ui.spinBoxGSig.value()
+        downsample_ratio = self.ui.spinBoxDownsample.value()
+        cnmfe_kwargs = \
+            {
+                'gSig':  (gSig, gSig)
+            }
+        d = dict()
+        d.update(
+            {
+                'do_cnmfe': False,
+                'cnmfe_kwargs': cnmfe_kwargs,
+                'downsample_ratio': downsample_ratio,
+            }
+        )
+
+        name = self.ui.lineEdCorrPNRName.text()
 
-        :return: parameters dict
-        :rtype: dict
-        """
+        self.add_item(name, d)
 
+    @present_exceptions()
+    def _add_item_cnmfe(self, *args, group_params: bool = True) -> Tuple[str, bool, dict]:
+        low_rank_background = self.ui.checkBox_low_rank_background.isChecked()
+        keep_memmap = self.ui.checkBoxKeepMemmap.isChecked()
         # CNMF kwargs
+        gSig = self.ui.spinBoxGSig.value()
+        kval = self.ui.spinBoxK.value()
+        if kval == 0:
+            kval = None
+        deconv = self.ui.comboBoxDeconv.currentText()
+        if deconv == 'SKIP':
+            method_deconvolution = None
+        else:
+            method_deconvolution = deconv
+        downsample_ratio = self.ui.spinBoxDownsample.value()
         cnmfe_kwargs = \
             {
+                'gSig': (gSig, gSig),
+                'gSiz': (4 * gSig + 1, 4 * gSig + 1),
```

Comment:
Ask Andrea if this is what's generally used, I forgot if it's generally `gSiz = 4 * gSig + 1` or `3 * gSig +1`

Reply:
Or just check the docs actually

Reply:
I got it from the demo doc :)

Reply:
haha, looks like I copied from the demo notebook too!